ultraLatency.py

```diff
@@ -198,6 +198,7 @@
         oh.setTriggerSource(ohboard,options.gtx,0x5) # GBT, 0x0 for GTX
         pass
 
+    oh.setTriggerThrottle(ohboard, options.gtx, options.throttle)
```

Comment:
maybe we should only write this if it is different from 1, which should be the default value. Just makes me feel better if we are not always writing this register.